File: src/main/webapp/resources/js/book.js
function submitBookForm() {
    var name = $('#name').val().trim();
    var code = $('#code').val();
    var price = $('#price').val();
    var authors = $('#authors').val().trim();
    var isbn = $('#isbn').val();
    var publisher = $('#publisher').val().trim();
    var publishedOn = $('#publishedOn').val();

    if (name.length == 0) {
        alert('Please enter name');
        $('#name').focus();
        return false;
    }

    if (code.length == 0 || code.length > 25) {
        alert('Please enter proper code');
        $('#code').focus();
        return false;
    }

    if (price.length == 0 || price.length > 5) {
        alert('Please enter proper price');
        $('#price').focus();
        return false;
    }

    if (authors.length == 0) {
        alert('Please enter authors');
        $('#authors').focus();
        return false;
    }

    if (isbn.length == 0 || isbn.length > 25) {
        alert('Please enter proper isbn');
        $('#isbn').focus();
        return false;
    }

    if (publisher.length == 0) {
        alert('Please enter publisher');
        $('#publisher').focus();
        return false;
    }

    if (publishedOn.length == 0) {
        alert('Please enter proper publishedOn');
        $('#publishedOn').focus();
        return false;
    }
    return true;
}

File: src/main/java/com/maxboguslavskyi/bookstore/entity/Book.java
package com.maxboguslavskyi.bookstore.entity;

import javax.persistence.*;
import java.io.Serializable;
import java.sql.Date;

@Entity
@Table(name = "books")
public class Book implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;

    @Column(name = "name")
    private String name;

    @Column(name = "code", length = 25)
    private String code;

    @Column(name = "price", length = 5)
    private String price;

    @Column(name = "authors")
    private String authors;

    @Column(name = "isbn", length = 25)
    private String isbn;

    @Column(name = "publisher")
    private String publisher;

    @Column(name = "published_date")
    private Date publishedOn;

    public Book() {
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }

    public String getAuthors() {
        return authors;
    }

    public void setAuthors(String authors) {
        this.authors = authors;
    }

    public String getIsbn() {
        return isbn;
    }

    public void setIsbn(String isbn) {
        this.isbn = isbn;
    }

    public String getPublisher() {
        return publisher;
    }

    public void setPublisher(String publisher) {
        this.publisher = publisher;
    }

    public Date getPublishedOn() {
        return publishedOn;
    }

    public void setPublishedOn(Date publishedOn) {
        this.publishedOn = publishedOn;
    }

    @Override
    public String toString() {
        return "Book {" +
                "id = " + id +
                ", name = '" + name +
                ", code = " + code +
                ", price = " + price +
                ", authors = " + authors +
                ", isbn = " + isbn +
                ", publisher = " + publisher +
                ", publishedOn = " + publishedOn +
                '}';
    }
}
